src/dataflow-types/src/client.rs

```diff
@@ -368,19 +368,32 @@ pub enum StorageCommand<T = mz_repr::Timestamp> {
 }
 
 /// A command that starts ingesting the given ingestion description
-#[derive(Arbitrary, Clone, Debug, Serialize, Deserialize, PartialEq)]
-pub struct IngestSourceCommand {
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct IngestSourceCommand<T> {
     /// The id of the storage collection being ingested.
     pub id: GlobalId,
     /// The description of what source type should be ingested and what post-processing steps must
     /// be applied to the data before writing them down into the storage collection
     pub description: IngestionDescription<CollectionMetadata>,
+    /// The upper frontier that this ingestion should resume at
+    pub resume_upper: Antichain<T>,
+}
+
+impl Arbitrary for IngestSourceCommand<mz_repr::Timestamp> {
+    type Strategy = BoxedStrategy<Self>;
+    type Parameters = ();
+
+    fn arbitrary_with(_: Self::Parameters) -> Self::Strategy {
+        panic!();
```

Comment:
I will fill this in concurrently with review. It's the reason `cargo test` fails in CI